### PR TITLE
Updated AzKubernetesV1SpecBuilder.java to add hostPath volumes

### PR DESCRIPTION
**Pull request by aditya1105**

### CODE REVIEW

File: azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java [hunk 1]
```diff
@@ -148,6 +148,28 @@ public AzKubernetesV1SpecBuilder addJobType(String name, String image, ImagePull
...
+     * container.
+     *
+     * @param volName      Name of the volume.
+     * @param hostPath     Path for the hostPath volume.
+     * @param hostPathType     Type for the hostPath volume.
+     * @param volMountPath Path to be mounted for the flow container.
+     */
+    public AzKubernetesV1SpecBuilder addHostPathVolume(String volName, String hostPath,
+        String hostPathType, String volMountPath) {
+        V1Volume hostPathVolume = new V1VolumeBuilder().withName(volName).withNewHostPath()
```
Comment (jakhani) [inline azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java:162]: Nit: final

File: azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java [hunk 2]
```diff
@@ -148,6 +148,28 @@ public AzKubernetesV1SpecBuilder addJobType(String name, String image, ImagePull
...
+            .withPath(hostPath).withType(hostPathType).endHostPath().build();
+        this.appVolumes.add(hostPathVolume);
+        V1VolumeMount hostPathVolMount = new V1VolumeMountBuilder()
```
Comment (jakhani) [inline azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java:165]: Nit: final

File: azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java (hunk 1 above)
Comment (aditya1105) [reply azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java]: done

File: azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java (hunk 2 above)
Comment (aditya1105) [reply azkaban-common/src/main/java/azkaban/container/models/AzKubernetesV1SpecBuilder.java]: done

**Review by jakhani — APPROVED** [review]
LGTM. Thank you for taking care of env variable setting.

**Pull request merged**